src/client/pages/IntiateCarCancellation/InitiateCarCancellationPage.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from 'react'
+import { Redirect, useLocation } from 'react-router'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import ReactMarkdown from 'react-markdown'
+import { useInitiateCarCancellationMutation } from 'data/graphql'
+import { LoadingDots } from 'components/LoadingDots/LoadingDots'
+import { Page } from 'components/utils/Page'
+import { ErrorHeading, ErrorModal, ErrorText } from 'components/ErrorModal'
+import { useTextKeys } from 'utils/textKeys'
+
+const CenteredWrapper = styled.div({
+  width: '100%',
+  height: '90vh', // 100 shows scroll bars since we have padding on parent elements
```

Comment:
If 10vh is less than paddings, UI may break.  Does it work on small mobile screen?

Reply:
Yes, it works on mobile screen. The only problem that might arise if 10vh is less than paddings is that we get a scroll bar on the page while the request to BE is loading, so I don't see that as a huge issue if it were to happen.